models/Element/Tag.php

```diff
@@ -55,6 +57,26 @@ class Tag extends Model\AbstractModel
      */
     public $parent;
 
+    /**
+     * @var string
+     */
+    public $translations;
```

Comment:
Maybe those properties should be `private` or `protected` because at least this one contains a serialized string and thus should be accessed through getters/setters.

Reply:
Totally aggree on this one, hence all models in the pimcore lifecyle have public properties, therefore I went with public as well

Reply:
yes, but we are in the process of changing them to protected ... so please go for protected for new ones

Reply:
You have no clue, how much this decission makes me happy.. there is a ton of problems that will just be gone with wind with this approach